srcpkgs/zathura/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'zathura'
 pkgname=zathura
-version=0.4.1
+version=0.4.3
 revision=1
 build_style=meson
-hostmakedepends="pkg-config intltool python3-Sphinx desktop-file-utils appstream-glib glib-devel"
+configure_args="-Dsynctex=disabled -Dtests=disabled"
+hostmakedepends="pkg-config intltool python3-Sphinx desktop-file-utils appstream-glib glib-devel librsvg-utils"
 makedepends="girara-devel sqlite-devel file-devel zlib-devel libseccomp-devel"
 short_desc="A highly customizable and functional document viewer"
 license="Zlib"
 maintainer="lemmi <lemmi@nerd2nerd.org>"
 homepage="http://pwmt.org/projects/zathura/installation/"
 changelog="https://pwmt.org/projects/zathura/changelog/${version}"
 distfiles="http://pwmt.org/projects/${pkgname}/download/${pkgname}-${version}.tar.xz"
-checksum=d6f56bf6e7e05d2e91213287c7aea21481c44521b622f9e9ec06a886f51dd9fe
+checksum=7e12194426d709670e0f4b0b107c80dd21322881b57d4a0bf9a09998402ffd41
 
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" glib-devel"
```

Comment:
no need to add to hostmakedepends conditionally if it is already there in the top declaration.